### PR TITLE
yoav - bag wrist

### PR DESCRIPTION
**Pull request by hillelv**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/utils/Conversions.java [hunk 1]
```diff
@@ -40,4 +40,15 @@ public static double convertRPMToMeterPerSecond(double rpm) {
...
 	}
+
+	public static class MagEncoderConversions {
+
+		public static Rotation2d MotorPositionToRotation2D(double motorPosition) {
+			return Rotation2d.fromRotations(motorPosition / 4096.0);
+		}
+
+		public static double Rotation2DToMotorPosition(Rotation2d angle) {
+			return angle.getRotations() * 4096.0;
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/utils/Conversions.java:51]: Maybe make a constant for this `4096.0`

**Review by yonnbro — APPROVED** [review]

**Comment by yonnbro** [comment]
Can you also rename to SRX or BrushedMotorWrist?

**Review by yonnbro — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/srxWrist/SRXWrist.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import edu.greenblitz.robotName.utils.Conversions;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class SRXWrist implements IWrist {
+
+    private TalonSRX motor;
+
+    public SRXWrist(){
+        motor = new TalonSRX(SRXWristConstants.MOTOR_ID);
+        motor.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Absolute, SRXWristConstants.PID_SLOT, SRXWristConstants.TIMEOUT_FOR_CONFIG_SET);
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/srxWrist/SRXWrist.java:20]:
```suggestion
        motor.configSelectedFeedbackSensor(
               FeedbackDevice.CTRE_MagEncoder_Absolute, 
               SRXWristConstants.PID_SLOT, SRXWristConstants.TIMEOUT_FOR_CONFIG_SET
        );
```

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/srxWrist/SRXWrist.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    @Override
+    public void setPower(double power) {
+        motor.set(TalonSRXControlMode.PercentOutput, power);
+    }
+
+    @Override
+    public void setVoltage(double voltage) {
+        motor.set(TalonSRXControlMode.PercentOutput, voltage / 12);
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/srxWrist/SRXWrist.java:31]: magic num

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/srxWrist/SRXWristConstants.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+    public static final int MOTOR_ID = 1;
+
+    public static final int PID_SLOT = 0;
+
+    public static final int TIMEOUT_FOR_CONFIG_SET = 0;
+
+    public static final double MAG_ENCODER_CONVERSION_FACTOR = 4096.0;
+
+    public static final TalonSRXConfiguration TALON_SRX_CONFIGURATION = new TalonSRXConfiguration();
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/srxWrist/SRXWristConstants.java:17]: add a space and delete the ones in the bottom

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/srxWrist/SRXWrist.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+import edu.greenblitz.robotName.subsystems.arm.wrist.IWrist;
+import edu.greenblitz.robotName.subsystems.arm.wrist.WristInputsAutoLogged;
+import edu.greenblitz.robotName.utils.Conversions;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class SRXWrist implements IWrist {
+
+    private TalonSRX motor;
+
+    public SRXWrist(){
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/srxWrist/SRXWrist.java:19]:
```suggestion
    public SRXWrist() {
```

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

**Review by bossyshrimp68 — APPROVED** [review]

**Pull request merged**